aws/ami/files/scylla_install_ami

```diff
@@ -85,6 +85,17 @@ if __name__ == '__main__':
             if not re.match(r'^ - mounts\n$', l):
                 f.write(l)
     run('/opt/scylladb/scripts/scylla_setup --ntp-domain amazon --no-coredump-setup --no-sysconfig-setup --no-raid-setup --no-io-setup --no-bootparam-setup --no-ec2-check --no-swap-setup')
+
+    # The difference comes from two different services to configure CPU scaling.
+    # On CentOS 'cpupower' fails when CPU scaling is not supported, so we can't
+    # enable it here.
+    # On Ubuntu 'cpufrequtils' never fails even CPU scaling is not supported,
+    # so we want to enable it here.
+    if distro == 'centos':
```

Comment:
@syuu1228 in 4.4 we don't have anything else then centos. so distro is not define. you can just drop line 94 and then the else.

Saw it after i merged. so i dropped the commit . please fix and then we will merge again

Reply:
@yaronkaikov Ahh, you are absolutely correct, just pushed CentOS-only version. Sorry about the mistake